src/cmd/sandbox.rs

```diff
@@ -146,6 +147,7 @@ pub struct SandboxBuilder {
     user: Option<String>,
     cmd: Vec<String>,
     enable_networking: bool,
+    image: Option<SandboxImage>,
```

Comment:
An idea I just had to avoid making `SandboxImage` cloneable is to just store the image name in `SandboxBuilder` as a `Option<String>`. What do you think?

Reply:
I have no opinion, if you'd like it to be changed I will do so.

Reply:
Please do!